packages/manager/modules/vps/src/dashboard/vps-dashboard.html

```diff
@@ -27,6 +27,23 @@
 <oui-message data-ng-if="$ctrl.vpsMigrationTaskInError" data-type="error">
     <p data-translate="vps_dashboard_migration_in_error"></p>
 </oui-message>
+<oui-message
+    data-ng-if="$ctrl.isVpsNewRange && !$ctrl.statistics"
+    data-type="info"
+>
+    <p
+        data-translate="vps_dashboard_rtm_missing"
+        data-translate-values="{
+      href: $ctrl.guide_rtm
```

Comment:
Indentation looks weird it. Could you please have a look?

Thank you!

Reply:
YEs, i edit it when i put the right link ;) thanks!